va/va.go

```diff
@@ -110,6 +124,13 @@ func New(
 		va.log.Printf("Disabling random VA sleeps")
 	}
 
+	sleepTime := os.Getenv(sleepTimeEnvVar)
+	sleepTimeInt, err := strconv.Atoi(sleepTime)
+	if err == nil && sleepTimeInt >= 1 {
```

Comment:
Sorry I missed this on the first pass but I think this `if` clause should also include `!noSleep` to ensure that the sleep time isn't overridden/logged when it won't be used because `noSleepEnvVar` was present.

Reply:
I added `!va.sleep`, since `noSleepEnvVar` can be present and have the value `0` (or `false` or whatever's not on the explicit truish list), in which case `noValidateEnvVar` still makes sense.